<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>549</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Kharif ET-Deficit Calculator</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>510</y>
     <width>156</width>
     <height>23</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="inputs_group_box">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>601</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Input</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QLineEdit" name="rainfall_csv_filename">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="rainfall_csv_browse">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="cadastral_layer_filename">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>140</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="lulc_layer_browse">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>140</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lulc_layer_filename">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="soil_layer_filename">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>90</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="zones_layer_filename">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>92</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="cadastral_layer_browse">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>140</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="soil_layer_browse">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>90</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QComboBox" name="crop_combo_box">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>240</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>175</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Land-Use-Land-Cover Vector Layer :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>72</y>
      <width>145</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Zones Vector Layer :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="zones_layer_browse">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>90</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>321</x>
      <y>72</y>
      <width>118</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Soil Cover Vector Layer :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>220</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Crop :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>220</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Daily Rainfall CSV File :</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>120</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Cadastral Map Vector Layer :</string>
    </property>
   </widget>
   <widget class="QPushButton" name="slope_layer_browse">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>190</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>98</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Slope Raster Layer :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="slope_layer_filename">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="folder_path_browse">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>40</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Data-set Folder :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="folder_path">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>491</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>170</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Drainage Vector Layer :</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="drainage_layer_filename">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>190</y>
      <width>191</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="drainage_layer_browse">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>190</y>
      <width>51</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Browse</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>220</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Sowing Threshold:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="sowing_threshold">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>240</y>
      <width>42</width>
      <height>22</height>
     </rect>
    </property>
    <property name="maximum">
     <number>1000</number>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>601</width>
     <height>181</height>
    </rect>
   </property>
   <property name="title">
    <string>Output</string>
   </property>
   <widget class="QGroupBox" name="save_image_group_box">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>120</y>
      <width>541</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Save As Image In</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <widget class="QPushButton" name="save_image_browse">
     <property name="geometry">
      <rect>
       <x>440</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>Browse</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="save_image_filename">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>321</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>46</width>
       <height>13</height>
      </rect>
     </property>
     <property name="text">
      <string>Location :</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>541</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Colour-code intervals for ET-Deficit map :</string>
    </property>
    <widget class="QPushButton" name="colour_code_intervals_split_button">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Splits the interval containing the selected value</string>
     </property>
     <property name="text">
      <string>&gt;&gt; Split</string>
     </property>
    </widget>
    <widget class="QPushButton" name="colour_code_intervals_merge_button">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>50</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="toolTip">
      <string>Merges with previous interval</string>
     </property>
     <property name="text">
      <string>Merge &lt;&lt;</string>
     </property>
    </widget>
    <widget class="QListWidget" name="colour_code_intervals_list_widget">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>20</y>
       <width>201</width>
       <height>51</height>
      </rect>
     </property>
     <property name="modelColumn">
      <number>0</number>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
     <property name="currentRow">
      <number>-1</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>40</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Split an interval at : </string>
     </property>
    </widget>
    <widget class="QSpinBox" name="colour_code_intervals_split_value_spin_box">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>40</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>271</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
